assets/config.js

```diff
@@ -314,3 +314,16 @@ export const regions = [
         boundaryData: null
     }
 ]
+
+export const reports = [
+    {
+        name: 'pdf',
+        label: 'PDF',
+        reportUrl: '/sst/create-pdf/'
```

Comment:
These can be just `url` since the `report` is implied. Just as you don't have `reportName` and `reportLabel`.